api/preprint_providers/views.py

```diff
@@ -75,13 +79,24 @@ class PreprintProviderList(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin
 
     ordering = ('name', )
 
-    # implement ODMFilterMixin
-    def get_default_odm_query(self):
-        return None
+    # implement DjangoFilterMixin
+    def get_default_django_query(self):
+        # TODO something more consistent: a filter[] param, or another endpoint (maybe under /users/me)
+        permission = self.request.GET.get('with_reviews_permission')
```

Comment:
I thought that might be the case... It seemed like that would be an annoying amount of work, but I might have been wrong. Now that I'm looking more closely at the filter mixins, they have some methods for custom filtering behavior, I can probably hook into them...